prow/gerrit/gerrit.go

```diff
@@ -294,14 +299,25 @@ func (c *Controller) ProcessChange(change gerrit.ChangeInfo) error {
 		if _, err := c.kc.CreateProwJob(pj); err != nil {
 			logger.WithError(err).Errorf("fail to create prowjob %v", pj)
 		} else {
-			triggered = append(triggered, spec.Name)
+			var b bytes.Buffer
+			if c.ca.Config().Plank.JobURLTemplate != nil {
+				if err := c.ca.Config().Plank.JobURLTemplate.Execute(&b, &pj); err != nil {
```

Comment:
There is a very unlikely race here. To be completely correct we should use the same `*config.Configuration` that we validate instead of fetching the config again in case the `JobURLTemplate` is removed in that time.

Reply:
done